operator/src/test/java/oracle/kubernetes/operator/helpers/PodHelperTestBase.java

```diff
@@ -247,7 +249,9 @@ public void setUp() throws Exception {
     WlsDomainConfigSupport configSupport = new WlsDomainConfigSupport(DOMAIN_NAME);
     configSupport.addWlsServer(ADMIN_SERVER, ADMIN_PORT);
     if (!ADMIN_SERVER.equals(serverName)) {
```

Comment:
I hadn't noticed this code before. Maybe it would be better to move it to the ManagedServer version of the setUp method rather than leaving it in the common code and testing the name.